src/ngraph/serializer.cpp

```diff
@@ -1891,7 +1901,15 @@ shared_ptr<Node> JSONDeserializer::deserialize_node(json node_js)
         case OP_TYPEID::Sum:
```

Comment:
Ditto comments for `Product`.

Reply:
Done
- pass `args[1]` when `reduction_axes` is empty (not serialized) for `v0::Sum`
- pass `args[1]` always for `v1::ReduceSum`